payjoin/src/receive/error.rs

```diff
@@ -172,6 +174,14 @@ impl fmt::Display for RequestError {
                     original_psbt_fee_rate, receiver_min_fee_rate
                 ),
             ),
+            InternalRequestError::FeeTooHigh(proposed_fee, max_fee) => write_error(
+                f,
+                "original-psbt-rejected",
+                &format!(
+                    "Receiver fee exceeds maximum allowed fee: {} > {}",
+                    proposed_fee, max_fee
+                ),
+            ),
```

Comment:
Shouldn't this error message reflect the parameterized feerate and the existing `PsbtBelowFeeRate` error?  If not, the error message doesn't really tell a user how their parameterized feerate induced the error and gives them little detail to potentially change it to a non-error outcome.